modules/misc/include/interfaces/GravityVectorInterface.h

```diff
@@ -0,0 +1,37 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InputParameters.h"
+#include "libmesh/vector_value.h"
+
+/**
+ * Interface for specifying gravity at the component level
+ */
+class GravityVectorInterface
```

Comment:
Great idea!

Reply:
both should be helpers / utils though

Reply:
I had started renaming (nothing destructive yet), but I'll wait for the final verdict by MOOSE naming judges ⚖️ If renaming to `GravityVectorHelper`, there is also the question of where it goes in the directory structure (`base`, `interfaces`, `utils`). I'd vote `utils`.

Reply:
If the methods are public then they are public interfaces